src/Raven.Server/Documents/Replication/ConflictManager.cs

```diff
@@ -85,9 +86,16 @@ public ConflictManager(DocumentDatabase database, ResolveConflictOnReplicationCo
                         conflicts.Add(local);
 
                     var resolved = _conflictResolver.ResolveToLatest(conflicts);
-                    _conflictResolver.PutResolvedDocument(documentsContext, resolved, conflictedDoc);
-
-                    return;
+                    try
+                    {
+                        _conflictResolver.PutResolvedDocument(documentsContext, resolved, conflictedDoc, localDocumentTuple.Document);
+                        return;
+                    }
+                    catch (AttachmentConflictException e)
+                    {
+                        if (_log.IsInfoEnabled)
```

Comment:
Under what scenario do we want this behavior? We should always resolve things automatically